src/main/scala/com/fortysevendeg/android/sqlite/metadata/SQLiteDatabaseMetaDataSupport.scala

```diff
@@ -0,0 +1,210 @@
+package com.fortysevendeg.android.sqlite.metadata
+
+import java.sql.{Connection, ResultSet, DatabaseMetaData}
+
+trait SQLiteDatabaseMetaDataSupport
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Number of methods in class exceeds 30](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/784801319/issues/source?bid=2442814&fileBranchId=2493741#l5)